### ISSUE TITLE
Social media image (og:image) for sub pages

### ISSUE THREAD

**Issue by zeichensatz** [benjaminkott/bootstrap_package#738]
# Support
TYPO3 v9.5.8
Bootstrap 11.0.0
SEO extension is activated

**Is your question related to a problem? Please describe.**
As far as I see, og:image will **_only_** be set if an image in the backend is related to the social media fields of the the specific page. The selected image will not be used for og:image on sub pages of this page.
That means I have to specify social media image field for EVERY page, which is not really convenient, if you have a (private) website with a several hundreds of pages :-(.
Is there any work around, that sub (or sub-sub...) pages will use og:image of a page higher in the directory tree?
Sorry if that's a stupid question. I'm not a TYPO3 professional.
Thanks a lot!

**Comment by gilbertsoft** [comment]
@haassie this sounds like a nice feature for the core or is this supported already somehow?

**Comment by haassie** [comment]
Well, to be honest I'm not a big fan of fallbacks of SEO related information because it doesn't make that much sense. Most of the times fallback information is not specific enough to give a visitor information about that specific page and doesn't help the visitor at all. In those cases you could better let the social media platform choose the information to show.

Same for images, although that would be the only field you can argue about in my opinion. That's why we added https://docs.typo3.org/c/typo3/cms-seo/master/en-us//Configuration/Index.html#setting-fallbacks-for-og-image-and-twitter-image to the documentation.

**Comment by zeichensatz** [comment]
Thanks for replying!
Just to explain my request: On my website I have many pages with a photo slider CE (provided by an own extension, which displays photo albums hosted on a Synology PhotoStation). There are no photos in other CEs on these pages.
Facebook (for example) does not discover any image information on these pages, until I set og:image/twitter-image (which I did for all pages in the meantime). Previous to the invention of the og-and-twitter-image the behaviour was different and it was possible to get a fallback image from the page tree. In my opinion the og:image is not such a specific information. It's just nice to get a image, which is somehow related to the topic of the page(s), if a link is posted in Facebook for example. And depending or your page tree structure it can make sense to use the same image for a lot of subpages.
Thanks to @haassie for mentioning the fallback solution, which might work in my case too, even though it points to an file not to a page in the page tree.
As I already solved my problem on the hard way, feel free to close this issue.
